clicommand/env_set.go

```diff
@@ -0,0 +1,169 @@
+package clicommand
+
+import (
+	"bufio"
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/buildkite/agent/v3/env"
+	"github.com/buildkite/agent/v3/jobapi"
+	"github.com/urfave/cli"
+)
+
+const envSetHelpDescription = `Usage:
+  buildkite-agent env set [variable]
+
+Description:
+   Sets environment variable values in the current job execution environment. 
+   Existing variables will be overwritten.
+
+   Note that this subcommand is only available from within the job executor with
+   the ′job-api′ experiment enabled.
+   
+   Note that changes to the job environment variables only apply to subsequent
+   phases of the job. To read the new values of variables from within the
+   current phase, use ′env get′.
+
+   Note that Buildkite read-only variables cannot be overwritten.
+
+Example (sets the variables ′LLAMA′ and ′ALPACA′):
+
+    $ buildkite-agent env set LLAMA=Kuzco "ALPACA=Geronimo the Incredible"
+	Added:
+	+ LLAMA
+	Updated:
+	~ ALPACA	
```

Comment:
I think there are some tabs here. Whereas indentation was with spaces in most of the other help texts.

Reply:
Should be fixed! (Curse you and your tab-mangling feature, modern text editors!)